services/parser-database/connector.py

```diff
@@ -20,20 +21,23 @@ def get_documents_to_parse():
 
 
 def get_keywords(text):
-    """Get keywords that relate to this article
+    """Get keywords that relate to this article (Javier's service)
 
     Args:
         text (sting): text to extract keywords from
 
     Returns:
         [list]: list of extracted keywords
     """
-    splited_text = text.split()
-    keywords = [
-        splited_text[random.randint(0, len(splited_text) - 1)],
-        splited_text[random.randint(0, len(splited_text) - 1)],
-    ]
-    return keywords
+
```

Comment:
You probably want to do a try/catch block around this whole block so if there are errors contacting the service or the response is missing important fields, you can error handle